amti/clis/associate.py

```diff
@@ -29,7 +29,7 @@
     '--qual', '-q', 
     help='QualificationId (or name, if --name flag passed).')
 @click.option(
-    '--value', '-v', 
+    '--integer_value', '-v', 
```

Comment:
Changing the flag name from `--integer_value` to `--integer-value` would make it more consistent with the other flag names in amti.